user_guide_src/source/concepts/factories.rst

```diff
@@ -334,14 +334,14 @@ Set the following property to ``true`` in **app/Config/Optimize.php**::
 
     public bool $configCacheEnabled = true;
 
-Since v4.5.0, you can enable this with the ``spark optimize`` command.
+Or you can enable this with the ``spark optimize`` command.
 
 .. note::
     This property cannot be overridden by
     :ref:`environment variables <configuration-classes-and-environment-variables>`.
 
 .. note::
-    Prior to v4.5.0, uncomment the following code in **public/index.php**::
+    Prior to v4.4.x, uncomment the following code in **public/index.php**::
```

Comment:
Why this change? I think "Prior to v4.5.0" is correct.
It means before v4.5.0 (4.4.x and before) and v4.5.0 is not included.

Reply:
Sorry my bad english. By the way, Config caching ready in version 4.4.x, i think better specify version `4.4.x`

Reply:
Ah, you mean "**In v4.4.x**", not "**Prior to v4.4.x**"?
Because Config caching was introduced in v4.4.0.

Reply:
Nah thats it